run_ci_minimization.sh

```diff
@@ -4,7 +4,7 @@
 
 set -ex
 
-if [ $# != 8 ]; then >&2 echo Bad argument count; fi
+if [ $# != 13 ]; then >&2 echo Bad argument count; fi
```

Comment:
Do you understand why the bad count was not preventing the rest of the script to run?

Reply:
This line just says "emit a message on stderr".  If we want the error to be fatal (do we?), we should add `; exit 1` inside the `if`

Reply:
I've added `exit 1` to both files